k8s/configmaps/front-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: front-config
+data:
+  DD_ENV: "prod"
+  DD_SERVICE: "front-edge"
```

Comment:
should this be `front-edge-configmap.yaml` in prevision of having both `front` and `front-edge` ?